package.json

```diff
@@ -68,5 +68,9 @@
   },
   "devDependencies": {
     "standard-version": "^8.0.0"
+  },
+  "dependencies": {
+    "@types/jest": "^26.0.20",
+    "react-redux": "^7.2.2"
```

Comment:
Surprised to see these additions. If `@types/jest` is required for making TypeScript tooling happy, then cool, but it's probably a dev dependency in that case.

Redux seems out of place.